exec/execPackTmp.sh

```diff
@@ -9,7 +9,7 @@ export AMI_TYPE=$4
 export RES_REPO="bldami_repo"
 export RES_AWS_CREDS="aws_bits_access"
 export RES_PARAMS="baseami_params"
-export SHIPPABLE_NODE_INIT_SCRIPT="ubu_14.04_docker_1.9.sh"
+export SHIPPABLE_NODE_INIT_SCRIPT="Ubuntu_14.04_Docker_1.9.sh"
```

Comment:
Hardcoding the values for stable/unstable ami's as we will not changes these values.